pkg/cmd/prebuild/add.go

```diff
@@ -21,10 +21,14 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	runOnAddFlag bool
+)
+
 var prebuildAddCmd = &cobra.Command{
-	Use:     "add",
+	Use:     "add [project-config]",
```

Comment:
Let's use `[PROJECT_CONFIG]` to be consistent with other commands

Reply:
updated the command